components/cli/pkg/commands/list_ingresses.go

```diff
@@ -314,6 +370,33 @@ func displayCellImageApisTable(cellImageContent *image.Cell) {
 	table.Render()
 }
 
+func getIngressValues(cellImageContent string) kubectl.Cell {
+	spinner := util.StartNewSpinner("Extracting Cell Image " + util.Bold(cellImageContent))
+	parsedCellImage, err := image.ParseImageTag(cellImageContent)
+	imageDir, err := ExtractImage(parsedCellImage, false, spinner)
+	if err != nil {
+		spinner.Stop(false)
+		util.ExitWithErrorMessage("Error occurred while extracting image", err)
```

Comment:
Return the error to the parent function instead of exiting from here. Do that in other places as well.

Reply:
Fixed